src/base/operations/initiate_kyc_recovery_builder.js

```diff
@@ -0,0 +1,44 @@
+import { default as xdr } from '../generated/xdr_generated'
+import { BaseOperation } from './base_operation'
+import { Keypair } from '../keypair'
+
+export class InitiateKYCRecoveryBuilder {
+  /**
+     * Creates operation to create KYC request
+     * @param {object} opts
+     * @param {string} opts.targetAccount - target account to initiate recovery for
+     * @param {string} opts.signer
+     * @returns {xdr.initiateKycRecoveryOp}
+     */
+  static initiateKycRecovery (opts) {
+    let attrs = {}
+
+    if (!Keypair.isValidPublicKey(opts.targetAccount)) {
+      throw new Error('opts.targetAccount is invalid')
+    }
+
+    if (!Keypair.isValidPublicKey(opts.signer)) {
+      throw new Error('opts.signer is invalid')
+    }
```

Comment:
here the validators too pls. However, the validators use the same `KeyPair.isValidPublickKey`, they also report a bit more descriptive message for you